test/extended/router/gatewayapicontroller.go

```diff
@@ -31,6 +36,12 @@ var (
 		configv1.ClusterVersionCapabilityOperatorLifecycleManager,
 	}
 )
+const (
```

Comment:
The gofmt shows that blank line is required before `const`, that's maybe why `ci/prow/verify` is failing.

Reply:
done